samwise.js

```diff
@@ -4,9 +4,7 @@
  * Copyright 2016 Henrik Ingo, henrik.ingo@avoinelama.fi
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?